README.md

```diff
@@ -144,6 +145,16 @@ To do so:
 
 Re-transcode is performed by Brightcove's Videocloud and takes few minutes. After it's done `Brightcove Video tech info` section will be updated.
 
+### Tencent
+
+TencentPlayer is used for videos hosted on tencent cloud.
+To test this player, you need to create a Tencent test account in the cloud.
+But to register for this service, you need to link a card and pass verification.
+The service testing was performed in the context of [commercial team's ticket](https://youtrack.raccoongang.com/issue/HARROW-976)
+and [product team's ticket](https://youtrack.raccoongang.com/issue/RGOeX-25789).
```

Comment:
please rephrase to emphasize that links are RG-internal (as we discussed on the standup)
```suggestion

Internal RG tickets with the testing outputs:
- [initial implementation and testing](https://youtrack.raccoongang.com/issue/HARROW-976)
- [RGOeX-25789](https://youtrack.raccoongang.com/issue/RGOeX-25789).
```